### PR TITLE
Fixed Mailjet template parameters

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

**Comment by Remg** [comment]
@illusionOfParadise, i need another review/approval after these changes

File: src/Mailjet/Message/CommitteeCitizenInitiativeNotificationMessage.php [hunk 1]
```diff
@@ -79,37 +76,28 @@ public static function create(array $recipients, CommitteeFeedItem $feedItem, st
     }
 
     private static function getTemplateVars(
+        string $recipientFirstName,
```
Comment (illusionOfParadise) [inline src/Mailjet/Message/CommitteeCitizenInitiativeNotificationMessage.php]: There is no need of this variable here, isn't it ?

File: src/Mailjet/Message/CommitteeCitizenInitiativeOrganizerNotificationMessage.php [hunk 2]
```diff
@@ -38,11 +38,11 @@ private static function getTemplateVars(
...
         return [
-            'referent_firstname' => self::escape($referentFirstName),
-            'referent_lastname' => self::escape($referentLastName),
-            'referent_contact_link' => $contactLink,
+            'animator_firstname' => self::escape($referentFirstName),
+            'animator_lastname' => self::escape($referentLastName),
+            'animator_contact_link' => $contactLink,
             'committee_name' => self::escape($committeeName),
-            'target_firstname' => self::escape($targetFirstName),
+            'prenom' => self::escape($targetFirstName),
```
Comment (illusionOfParadise) [inline src/Mailjet/Message/CommitteeCitizenInitiativeOrganizerNotificationMessage.php]: There is no need of this variable here, isn't it ?
You should create `getRecipientVars` for it.

File: src/Mailjet/Message/CommitteeCitizenInitiativeNotificationMessage.php (hunk 1 above)
Comment (Remg) [reply src/Mailjet/Message/CommitteeCitizenInitiativeNotificationMessage.php]: good catch

**Pull request merged**